packages/common/src/TruffleConfig.js

```diff
@@ -176,6 +177,7 @@ addLocalNetwork(networks, "test");
 // Mainnet fork is just a local network with id 1 and a hardcoded gas limit because ganache has difficulty estimating gas on forks.
 // Note: this gas limit is the default ganache block gas limit.
 addLocalNetwork(networks, "mainnet-fork", { network_id: 1, gas: 6721975 });
+addLocalNetwork(networks, "polygon-matic-fork", { network_id: 137, gas: 6721975, gasPrice: 1000000000 });
```

Comment:
Just out of curiosity, why is a hardcoded gas price required for the fork network?

Reply:
Its not , we can remove!

Reply:
I left it there on principle

Reply:
Ah, I commented this before I saw the other hard coded gas prices.